core/tests/protractor/editorAndPlayer.js

```diff
@@ -26,17 +25,26 @@ var CollectionEditorPage =
   require('../protractor_utils/CollectionEditorPage.js');
 var CreatorDashboardPage =
   require('../protractor_utils/CreatorDashboardPage.js');
+var ExplorationEditorPage =
+  require('../protractor_utils/ExplorationEditorPage.js');
+var ExplorationEditorSettingsTab = require(
+  '../protractor_utils/ExplorationEditorSettingsTab.js');
 var ExplorationPlayerPage =
   require('../protractor_utils/ExplorationPlayerPage.js');
 var LibraryPage = require('../protractor_utils/LibraryPage.js');
 
 describe('Full exploration editor', function() {
   var explorationPlayerPage = null;
+  var explorationEditorPage = null;
+  var explorationEditorSettingsTab = null;
   var creatorDashboardPage = null;
   var libraryPage = null;
 
   beforeAll(function() {
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorSettingsTab =
+    new ExplorationEditorSettingsTab.ExplorationEditorSettingsTab();
```

Comment:
Indentation should be like this:

```
    explorationEditorSettingsTab = (
      new ExplorationEditorSettingsTab.ExplorationEditorSettingsTab());
```

Reply:
Though I think this should be more like:

```
    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
```

I.e. this test should only have access to explorationEditorPage, and explorationEditorPage has access to all its components.

Reply:
Fixed!

Reply:
Done !